keras_cv/layers/preprocessing3d/base_augmentation_layer_3d.py

```diff
@@ -18,7 +18,10 @@
 BOUNDING_BOXES = "bounding_boxes"
 OBJECT_POINT_CLOUDS = "object_point_clouds"
 OBJECT_BOUNDING_BOXES = "object_bounding_boxes"
+ADDITIONAL_POINT_CLOUDS = "additional_point_clouds"
+ADDITIONAL_BOUNDING_BOXES = "additional_bounding_boxes"
```

Comment:
Isn't additional point clouds the same as object point clouds? I was under the impression that group_points_by_boxes in only called for the additional point clouds

Reply:
Yes, OBJECT_POINT_CLOUDS are extracted from ADDITIONAL_POINT_CLOUDS.

all_point_clouds = [point_clouds, additional_point_cloud]

We use group_by function to extract objects from additional_point_cloud for copy-paste data augmentation.

Reply:
I think given the complexity of the relationship between e.g. POINT_CLOUDS, OBJECT_POINT_CLOUDS, and ADDITIONAL_POINT_CLOUDS, we should add some documentation in the base layer about what they mean and how to use them.

(In another PR is fine with me)

Reply:
Agree. I will document those in a different PR once we check in all the augmentation layers.